velox/exec/RowContainer.h

```diff
@@ -320,11 +340,11 @@ class RowContainer {
   /// a row with uninitialized keys for aggregates with no-op partial
   /// aggregation.
   void setAllNull(char* row) {
+    removeOrUpdateRowColumnStats(row, /*setToNull=*/true);
```

Comment:
Do we need to call `removeOrUpdateRowColumnStats` here? I saw `setAllNull` is only used for a new row in https://github.com/facebookincubator/velox/blob/f3585b13ff41b7cc64167a351bbc2a7d5b720b65/velox/exec/GroupingSet.cpp#L1345

Reply:
It breaks the related unit tests. So I guess although it is only used in new row for now, the assumption is that it can be used in other places?

Reply:
Good point. We actually should implement setAllNull as add null for all the columns involved.